File: 02.PYTHON/pg-166-169/pg_166_08.py
print("");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPY");
print("READ A TWO-DIGITS INTEGER AND DETERMINE IF ITS DIGITS ARE PRIMES");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPY");
print("");

num = int(input("Write a two-digits integer: "))
prime1 = True;
prime2 = True;
dig1 = 0;
dig2 = 0;

if(num < 0):
    num *= -1;

if(num >= 10 and num <= 99):
    dig2 = num % 10;
    dig1 = num // 10;
    print(dig1, "  ", dig2);
    
    i = 2;
    j = 2;
    for i in range(i, dig1):
        if(dig1 % i == 0):
            prime1 = False;
            break;

    for j in range(j, dig2):
        if(dig2 % j == 0):
            prime2 = False;
            break;
    
    if(prime1):
        print("The first digit is prime ");
    else:
        print("The first digit isn't prime ");
        

    if(prime2):
        print("and the second digit is prime. ");
    else:
        print("and the the second digit isn't prime. ");

else:
    print("The written number doesn't have two digits. Please try again!");

File: 01.JAVA/pg-166-169/pg_166_08.java
import java.util.Scanner;

public class pg_166_08
{
    public static void main(String[]args)
    {
        Scanner enter = new Scanner(System.in);
        int num, dig1, dig2, i, j;
        boolean prime1 = true;
        boolean prime2 = true;


        System.out.println("");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJV");
        System.out.println("READ A TWO-DIGITS INTEGER AND DETERMINE IF ITS DIGITS ARE PRIMES");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJV");
        System.out.println("");

        System.out.print("Write a two-digits integer: ");
        num = enter.nextInt();

        if(num < 0)
        {
            num *= -1;
        }

        if(num >= 10 && num <= 99)
        {
            dig2 = num % 10;
            dig1 = num / 10;

            System.out.println(dig1 + "  " + dig2);

            for(i = 2; i < dig1; i++)
            {
                if(dig1 % i ==  0)
                {
                    prime1 = false;
                    break;
                }
            }

            for(j = 2; j < dig2; j++)
            {
                if(dig2 % j == 0)
                {
                    prime2 = false;
                    break;
                }
            }

            if(prime1)
            {
                System.out.println("The first digit is prime ");
            }
            else
            {
                System.out.println("The first digits isn't prime ");
            }

            if(prime2)
            {
                System.out.println("and the second digit is prime ");
            }
            else
            {
                System.out.println("and the second digits isn't prime ");
            }
        }
        else
            {
                System.out.println("The written number doesn't have two digits. Please try again!");
            }
    }
}